crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__test__schema_sdl_export.snap

```diff
@@ -645,9 +631,14 @@ type TransactionBlockEdge {
 type TransactionBlockEffects {
 	status: ExecutionStatus!
 	errors: String
+	lamportVersion: Int
```

Comment:
Not sure if this the right implementation for lamport version. Can we please double check?

Reply:
What the issue?
Lamport timestamp is a u64.

Reply:
Originally I used `BigInt` for all fields whose types could technically exceed 2^53, but when we wrote the schema, we did some back of the envelope calculations and determined that we wouldn't see an object with a lamport version that high for 100s or even 1000s of years, so as a practical measure we made this an `Int`.